src/WPUpdatePhp.php

```diff
@@ -47,9 +82,17 @@ private function load_minimum_required_version_notice() {
 		}
 	}
 
+	/**
+	 * Generate admin notice
+	 */
 	public function admin_notice() {
-		echo '<div class="error">';
-		echo '<p>Unfortunately, this plugin can not run on PHP versions older than '. $this->minimum_version .'. Read more information about <a href="http://www.wpupdatephp.com/update/">how you can update</a>.</p>';
-		echo '</div>';
+		?>
+		<div class="error">
+			<p>
+				<?php printf( __( 'Unfortunately, %1$s can not run on PHP versions older than %2$s. Read more information about %3$sshow you can update%4$s.' ), $this->plugin_name, $this->minimum_version, '<a href="http://www.wpupdatephp.com/update/">', '</a>' ); ?>
```

Comment:
I would make the url translatable, too.
Translator can set something like `http://www.wpupdatephp.com/de/update/` or whatever the syntax for different languages for the site will be.